README.md

```
@@ -38,6 +40,9 @@ This approach aims to:
 
 ## Project Structure
 
+### [Packages](./packages/README.md)
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)